Make the start command print the endpoint on stdout (#1271)
tenzir/vast: CHANGELOG.md

## Unreleased

> - ⚠️ `vast start` now prints the endpoint it is listening on on `stdout` when it
>   is invoked with the `--print-endpoint` option.

Reviewer: ```suggestion
- ⚠️ `vast start` now prints on `stdout` the endpoint it is listening on,
  provided the user added the `--print-endpoint` option.
```

Author: I took the liberty to remove "on `stdout`".
